client/my-sites/exporter/export-card/index.jsx

```diff
@@ -42,7 +42,7 @@ class ExportCard extends Component {
 				loading={ this.props.shouldShowProgress }
 				isPrimary={ false }
 				onClick={ this.props.exportAll }
-				text={ translate( 'Export All' ) }
+				text={ translate( 'Export all' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.